examples/screenshot-spirv.rs

```diff
@@ -0,0 +1,63 @@
+#[macro_use]
+extern crate glium;
+
+#[allow(unused_imports)]
+use glium::{glutin, Surface};
+use glium::index::PrimitiveType;
+
+fn main() {
+    // building the display, ie. the main object
+    let event_loop = glutin::event_loop::EventLoop::new();
+    let wb = glutin::window::WindowBuilder::new().with_visible(true);
+    let cb = glutin::ContextBuilder::new();
+    let display = glium::Display::new(wb, cb, &event_loop).unwrap();
+
+    // building the vertex buffer with no vertices
+    let vertex_buffer = {
+        #[derive(Copy, Clone)]
+        struct Vertex {
+            dummy: f32,
+        }
+
+        implement_vertex!(Vertex, dummy);
+
+        glium::VertexBuffer::<Vertex>::empty(&display, 0).unwrap()
+    };
```

Comment:
@est31 Is there a more idiomatic way to create an empty `VertexBuffer` (without having to define a `Vertex` type)?

Reply:
Not that I knew. All the examples I've seen do define a vertex type.